src/adservice/Dockerfile

```diff
@@ -11,9 +11,14 @@ COPY . .
 RUN chmod +x gradlew
 RUN ./gradlew installDist
 
-FROM openjdk:8-alpine
-
-RUN apk add --no-cache libc6-compat
+FROM openjdk:8-slim
+
+# Download Stackdriver Profiler Java agent
+RUN apt-get update
+RUN apt-get install -qq wget
```

Comment:
please do these

1. use -y to update
2. use -y to install as well
3. merge RUN steps so the intermediate resuls don't get cached
4. chain a step to remove the apt cache.

an example of this would look like:

```
RUN apt-get -y update && apt-get install -qqy \
    pkg-1 \
    pkg-2 \
    pkg-3
 && rm -rf /var/lib/apt/lists/*
```

Reply:
done.